openapi/openapi.yaml

```diff
@@ -1120,6 +1120,36 @@ paths:
               schema:
                 $ref: "#/components/schemas/AnalysisResponseError"
 
+  /python/packages:
+    get:
+      tags: [PythonPackages]
+      x-openapi-router-controller: thoth.user_api.api_v1
+      operationId: list_python_packages
+      summary: >
+        Retrieve information from the Knowledge Graph with regards
+        to Python Package Names and the number of distinct Versions.
+      parameters:
+        - name: page
+          in: query
+          required: false
+          description: Page offset in pagination.
+          schema:
+            type: integer
+            default: 0
+      responses:
+        "200":
+          description: A list of Python Package Names and the number of distinct Versions.
```

Comment:
Not sure about capitals for **P**ackage, **N**ames and **V**ersions. Are they needed?